server/plugin/plugin.go

```diff
@@ -250,6 +263,86 @@ func (p *Plugin) OnDeactivate() error {
 	return nil
 }
 
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	githubEmoji := p.emojiMap[reaction.EmojiName]
+	if githubEmoji == "" {
+		return
+	}
+	post, error := p.client.Post.GetPost(reaction.PostId)
+	if error != nil {
+		p.API.LogError("Invalid post id. Addition of reaction failed.")
+		return
+	}
+
+	repo := post.GetProp("gh_repo")
+	if repo == nil {
+		p.API.LogDebug("Required prop is not present. This post is not created by this plugin, hence ignoring")
+		return
+	}
+
+	orgRepo := strings.Split(repo.(string), "/")
+	// strange that gh_object_id is always returned as float64, without type casting it throws an error
+	id := int(post.GetProp("gh_object_id").(float64))
```

Comment:
This sort of comment on the code "strange that gh_object_id..." should be in a comment in the PR discussion rather than a code comment

Reply:
```suggestion
	objectIDFloat, ok := post.GetProp("gh_object_id").(float64)
	if !ok {
	    // maybe just return without logging?   
	}
	
	objectID := int64(objectIDFloat)
```

`int64` seems like the right type to use since we end up casting to that later anyway